lib/src/cloudinary_response.dart

```diff
@@ -27,6 +29,7 @@ class CloudinaryResponse {
       url: data['url'],
       secureUrl: data['secure_url'],
       originalFilename: data['original_filename'],
+      tags: (data['tags'] as List).map((tag) => tag as String).toList()
```

Comment:
Good work. However you need check if `data['tags']` is null here

Reply:
Right, if data['tags'] is null then the property is set to an empty list. Or do you prefer to be null?

Reply:
Casting null to List will give you null then `.map()` will throw an exception.

`The method 'map' was called on null.
Receiver: null
Tried calling: map<String>(Closure: (dynamic) => String)`